multicompany_property_base/models/res_partner.py

```diff
@@ -55,7 +61,9 @@ def _children_sync(self, values):
                 if fields_to_sync:
                     vals = {f: p_val.get(f) for f in fields_to_sync}
                     self.with_context(force_company=company_id)._children_sync(vals)
+                values_to_upd.update(p_val)
             values.pop("property_ids")
+            values.update(values_to_upd)
```

Comment:
if you are putting the values to update on the super, is the call to _children_sync on line 63 needed?

Reply:
I've refactored it so that I pass the sanitized values to the super call with the respective force company context of the property being changed. So yes, the extra `_children_sync` call wasn't necessary.